clang/include/clang/AST/Type.h

```diff
@@ -4699,16 +4699,16 @@ class FunctionEffect {
 
 private:
   LLVM_PREFERRED_TYPE(Kind)
-  unsigned FKind : 3;
+  uint8_t FKind : 3;
```

Comment:
I mean, if this is the only field, then can’t we just make the actual type of this `Kind` since that’s a `uint8_t` enum anyway? Or is it likely that we’ll be adding more fields to this soon?

Reply:
Good point. There is a comment anticipating what might happen next with the class but it's hypothetical. For now there's no need for a bitfield.